src/test/java/fr/inria/diversify/automaticbuilder/GradleAutomaticBuilderTest.java

```diff
@@ -29,39 +36,85 @@
 
     @Before
     public void setUp() throws Exception {
+
+        cleanTestEnv();
+
+        Utils.init("src/test/resources/sample/sample.properties");
```

Comment:
You must init Utils with the right properties file: src/test/resources/test-projects/test-projects.properties